sbi/inference/snpe/snpe_base.py

```diff
@@ -353,7 +366,12 @@ def _train(
         self._summary["best_validation_log_probs"].append(self._best_val_log_prob)
 
     def _loss(
-        self, theta: Tensor, x: Tensor, masks: Tensor, calibration_kernel: Callable,
+        self,
+        theta: Tensor,
+        x: Tensor,
+        masks: Tensor,
+        proposal: Any,
```

Comment:
If proposal is not passed, it will automatically use `proposal=prior` and run single round (i.e. maximum likelihood loss). So, there is no scenario `proposal=None` **and** non-atomic loss.

This was part of the major multi-round API change a couple of weeks ago -- maybe it overlapped with your vacation, I'm happy to chat about it :)

Reply:
OK, it's fine, thanks!